### DISCUSSION TITLE
archlinux下报错

### DISCUSSION THREAD

**Discussion by ninjaWw** [tw93/Pake#373] [General]
$ pake https://weekly.tw93.fun --name weekly --transparent

To build the Linux app, you need to install Rust and Linux package
See more in https://tauri.app/v1/guides/getting-started/prerequisites#installing

You haven't provided an app icon, so the default icon will be used. To assign a custom icon, please use the --icon option.
npm verb cli /usr/bin/node /usr/bin/npm
npm info using npm@9.6.2
npm info using node@v19.8.1
npm verb title npm install
npm verb argv "install" "--loglevel" "verbose"
npm verb logfile logs-max:10 dir:/home/openwrt/.npm/_logs/2023-03-28T18_03_47_206Z-
npm verb logfile /home/openwrt/.npm/_logs/2023-03-28T18_03_47_206Z-debug-0.log
npm verb reify failed optional dependency /home/openwrt/.npm-global/lib/node_modules/pake-cli/node_modules/fsevents
npm http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 1162ms

up to date, audited 237 packages in 1s

78 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm verb exit 0
npm info ok 

> pake-cli@1.3.0 build
> npm run tauri build --release

> pake-cli@1.3.0 tauri
> tauri build

       Error `tauri.conf.json` error on `tauri > bundle > targets`: ["all"] is not valid under any of the given schemas
file:///home/openwrt/.npm-global/lib/node_modules/pake-cli/dist/cli.js:1828
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///home/openwrt/.npm-global/lib/node_modules/pake-cli/dist/cli.js:1828:24
    at /home/openwrt/.npm-global/lib/node_modules/pake-cli/node_modules/shelljs/src/exec.js:145:9
    at ChildProcess.exithandler (node:child_process:427:5)
    at ChildProcess.emit (node:events:512:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:304:5)

Node.js v19.8.1

请教下，在archlinux下出现的这个问题，应该怎么解决？

**Comment by Tlntin** [comment]
重复，参加该帖https://github.com/tw93/Pake/discussions/360
